### PR TITLE
[Share to IG] QA - Pick most colorful colors and adjust the wavform

### PR DESCRIPTION
**Pull request by nicoback2**
### Description
Demos/discussion:
https://audius-internal.slack.com/archives/C04885C9WNA/p1670778634084659

-Pick the two most colorful colors for the video
-Mirror the showfreqs wavform and make it transparent
### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

File: packages/mobile/threads/dominantColors.thread.js [hunk 1]
```diff
@@ -56,6 +56,28 @@ const findIndexOfMaxEuclideanDistance = (existing, selectFrom) => {
...
+}
+
+/**  Picks the two most dominant colors that meet a min threshold of colorfulness. */
+export const pickTwoMostDominantAndColorful = (colors) => {
+  const filtered = colors.filter((c) => {
+    return calculateColorfulness(c) >= 50.0
+  })
+  if (filtered.length < 2) {
+    return colors
+      .sort((a, b) => {
```
Comment (dylanjeffers) [inline packages/mobile/threads/dominantColors.thread.js:73]: Does sort return the array?

File: packages/mobile/threads/dominantColors.thread.js [hunk 2]
```diff
@@ -56,6 +56,28 @@ const findIndexOfMaxEuclideanDistance = (existing, selectFrom) => {
...
+const calculateColorfulness = (color) => {
+  // Reference: https://stackoverflow.com/questions/61705774/mathematically-calculate-vibrancy-of-a-color
+  const max = Math.max(color.r, color.g, color.b)
+  const min = Math.min(color.r, color.g, color.b)
+  return ((max + min) * (max - min)) / max
+}
+
+/**  Picks the two most dominant colors that meet a min threshold of colorfulness. */
+export const pickTwoMostDominantAndColorful = (colors) => {
+  const filtered = colors.filter((c) => {
```
Comment (dylanjeffers) [inline packages/mobile/threads/dominantColors.thread.js:68]: Nit but maybe, vibrantColors

File: packages/mobile/threads/dominantColors.thread.js [hunk 3]
```diff
@@ -56,6 +56,28 @@ const findIndexOfMaxEuclideanDistance = (existing, selectFrom) => {
   return indexOfMaxDistance
 }
 
+const calculateColorfulness = (color) => {
```
Comment (dylanjeffers) [inline packages/mobile/threads/dominantColors.thread.js:59]: Calc vibrancy?

**Review by dylanjeffers — APPROVED** [review]
Sweet

File: packages/mobile/threads/dominantColors.thread.js (hunk 1 above)
Comment (nicoback2) [reply packages/mobile/threads/dominantColors.thread.js:73]: yeee

**Pull request merged**